### DISCUSSION TITLE
Adding to collections shouldn't scroll me up

### DISCUSSION THREAD

**Discussion by ebolite** [ppy/osu#24312] [Ideas] [answered]
I have a ton of collections to sort my maps with. The issue is every time I add a map to a collection, it scrolls all the way back up to the top for some reason. It would be cool if it didn't do that so I could easily put a map in multiple collections.

https://github.com/ppy/osu/assets/22662047/c507ba15-ef4e-4a97-95eb-98fec640b214

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/24315
